SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2336,6 +2357,27 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 void USpatialNetDriver::ProcessRemoteFunction(AActor* Actor, UFunction* Function, void* Parameters, FOutParmRec* OutParms, FFrame* Stack,
 											  UObject* SubObject)
 {
+	FString WorkerId = [&]() -> FString {
+		if (World == nullptr)
+		{
+			return "(no world)";
+		}
+
+		const UGameInstance* GameInstance = World->GetGameInstance();
+		if (GameInstance == nullptr)
+		{
+			return "(no game instance)";
+		}
+
+		return GameInstance->GetSpatialWorkerId();
+	}();
+	FString FunctionName = Function->GetName();
+
+	if (FunctionName == "ServerSetEnabled")
+	{
+		UE_LOG(LogTemp, Warning, TEXT("[%s] ProcessRemoteFunction %s %s"), *WorkerId, *Function->GetName(), *Actor->GetFullName());
```

Comment:
we always hit this when the client tries to call a server RPC, regardless of whether it will make it through